client/src/components/JobCard/JobCard.js

```diff
@@ -25,6 +27,11 @@ class JobCard extends React.PureComponent {
       <div className="JobCard__Content">
         <CategoryIcon category={ this.props.category } />
         { this.props.name }
+        <span className="JobCard__Links">
+          <a href={jenkinsLink}>
```

Comment:
Should we add `rel="noopener noreferrer" target="_blank" to open it in a separate tab?